src/modules/todo/addTask.js

```diff
@@ -0,0 +1,106 @@
+import task from './todo';
```

Comment:
## Fix the `addTodo` task 
 - [ ] A user can not add the task as is required of this project. we are currently getting an error at the console.